docs/src/pages/docs/functions/combinators.md

```diff
@@ -309,6 +309,90 @@ with this library. It quite simply is just a function that when you pass it
 something, it returns that thing right back to you. So simple, I will leave it
 as an exercise to reason about why this is so powerful and important.
 
+#### psi
```

Comment:
for the other combinators, we've in the past prefered descriptive names over conventional ones. (i.e. 'flip' vs 'kestrel' or 'C')

in this case, maybe we'd prefer 'on' ?

Reply:
Yeah i know `on` is what this is called in Haskell and Sanctuary but i don't find it to be that descriptive honestly, I got the original reference and name from [here](https://gist.github.com/Avaq/1f0636ec5c8d6aed2e45) we seem to have matched the names with most everything else except `thrush`

Reply:
looks like a solid precedent. Thanks for addressing this.